lms/djangoapps/discussion_api/api.py

```diff
@@ -276,7 +286,12 @@ def get_thread_list(request, course_key, page, page_size, topic_id_list=None, te
         "per_page": page_size,
         "text": text_search,
     }
+
     text_search_rewrite = None
+
+    if view:
+        query_params[view] = "true"
```

Comment:
It's good that the HTTP API handles that case correctly, but I still think it would be better to raise an exception here. If someone provides an incorrect value, it's probably the result of a coding error that should be caught as early as possible.